en/ch03.md

```diff
@@ -1,17 +1,58 @@
+After creating and opening a wallet file, Monero GUI will start synchronizing. This process is required to know your balance, receive and send transactions.
 
-(1) **Balance All:** Let you see the sum of all your accounts balances, unlocked or not.  
-(2) **Accounts:** This is the list of all accounts already generated on your wallet and their corresponding account addresses.  
-(3) **Create new account:** Click on this button to create a new account and set its *optional* label.
+Monero GUI's synchronization is composed of two parts:
+1) Daemon synchronization
+2) Wallet synchronization
 
-You can change the current account by simply clicking on it on the list.  
-Each of your accounts are filled with their own informations:
+## Daemon synchronization
+Monero GUI wallet uses a daemon (monerod), which is a program running in the background that synchronizes with the Monero network to scan for incoming transactions and to send new transactions.
 
-(4) **Label:** A name or description of the account for convenience.  
-(5) **Account Address:** This is the public address you should give to the sender when you want to receive money in this account. In account #0, it starts with 4 and corresponds to the wallet's primary address. In the remaining accounts it corresponds to the account's first subaddress and therefore starts with 8.  
-(6) **Balance:** The account total balance. It cumulates the main address and all subaddresses for this account. *Note: Subaddresses are managed in the [Receive](#receive-monero) screen*  
-(7) **Edit Label:** Click on this button to edit this account label.  
-(8) **Copy Address to Clipboard:** Click on this button to copy this account address to the clipboard.  
+Your daemon must have access to the Monero blockchain, which is a very large file (more than 100 GB) which contains all transactions that have been made since the beginning of Monero in 2014. The blockchain file can be stored in your computer (if your daemon run a local node) or in a computer of another person (if your daemon connect to a third-party remote node).
+
+### Daemon synchronization with local node
+If you use a local node (full node), your wallet daemon will download the blockchain file into your computer. Since it's a very large file (more than 100 GB), this can take a long time, even days, depending on your internet speed.
+
+If you use a local node with a bootstrap node or "simple mode (bootstrap)", your wallet daemon will also download the blockchain file into your computer, but it will connect to a remote node (a bootstrap node) while the download is not finished.
```

Comment:
```suggestion
If you use a local node with a bootstrap node or "simple mode (bootstrap)", your wallet daemon will also download the blockchain file into your computer, but it will connect to a remote node (a bootstrap node) while the download is not finished, allowing you to immediately send/receive a transaction.
```

Reply:
I agree